.github/workflows/tests.yml

```diff
@@ -74,7 +74,7 @@ jobs:
       - name: Test with pytest
         id: tests
         run: |
-          python -m pytest -n auto --ignore=tests/benchmarks --cov=histolab
+          python -m pytest --ignore=tests/benchmarks --cov=histolab
```

Comment:
mm why?

Reply:
pthon 3.8 and the latest version of python xdist has some problems ... so we decide to get rid of that option for the moment